src/dotnet/APIView/APIView/Model/TokenTreeModel.cs

```diff
@@ -158,43 +252,113 @@ public static StructuredToken CreateStringLiteralToken(string value)
 
     public class APITreeNode
     {
-        public HashSet<string> Tags
+        /// <summary>
+        /// Id of the node, which should be unique at the node level. i.e. unique among its siblings.
+        /// This was previously represented by the DefinitionId for the main Token of the node.
```

Comment:
You can go with either of the examples above and be fine as far as the APITreeNodes are concerned. However, you need to make sure to use the same ID used for tokens that should allow navigation to the node. And since those tokens also need to support existing APIView comments, you should just use the same style used in your existing parser i.e. Foo.Foo.bar